app/assets/stylesheets/components/concept_details.scss

```diff
@@ -1,6 +1,6 @@
 .concept_details_component .raw-table .dropdown-title-bar p {
-  color: rgb(136, 136, 136);
-  font-weight: 400;
+  color: #656363;
```

Comment:
you can use the primary color, here also

![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/3e516dbe-2ea0-482c-bcae-7e09845b8fdf)